scripts/fix-next-installation-enhanced.js

```
@@ -0,0 +1,247 @@
+ * It addresses the problem where binary files aren't properly created,
+ * causing build failures even after running the fix:next script.
+ *
+ * The script:
+ * 1. Detects the operating system
+ * 2. Checks for existing Next.js installation
+ * 3. Removes problematic files
+ * 4. Reinstalls Next.js with proper binary creation
+ * 5. Verifies the installation
+ */
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)